diff --git a/vendor/automattic/jetpack-config/src/class-config.php b/vendor/automattic/jetpack-config/src/class-config.php
index 8d275d68b3..7f6e2c3cd9 100644
--- a/vendor/automattic/jetpack-config/src/class-config.php
+++ b/vendor/automattic/jetpack-config/src/class-config.php
@@ -263,6 +263,18 @@ class Config {
 		return true;
 	}

+	/**
+	 * Setup the Sync options.
+	 */
+	protected function ensure_options_sync() {
+		$options = $this->get_feature_options( 'sync' );
+		if ( method_exists( 'Automattic\Jetpack\Sync\Main', 'set_sync_data_options' ) ) {
+			Sync_Main::set_sync_data_options( $options );
+		}
+
+		return true;
+	}
+
 	/**
 	 * Temporary save initialization options for a feature.
 	 *
