diff --git a/client/overview/task-list/tasks.tsx b/client/overview/task-list/tasks.tsx
index 8b55aa86b..466f3c849 100644
--- a/client/overview/task-list/tasks.tsx
+++ b/client/overview/task-list/tasks.tsx
@@ -70,18 +70,16 @@ export const getTasks = ( {
 		0 < getDisputesDueWithinDays( activeDisputes, 7 ).length;
 
 	return [
-		showUpdateDetailsTask &&
-			! isPoEnabled &&
-			getUpdateBusinessDetailsTask(
-				errorMessages,
-				status ?? '',
-				accountLink,
-				Number( currentDeadline ) ?? null,
-				pastDue ?? false
-			),
-		wpcomReconnectUrl && getReconnectWpcomTask( wpcomReconnectUrl ),
-		isDisputeTaskVisible && getDisputeResolutionTask( activeDisputes ),
-		isPoEnabled && getVerifyBankAccountTask(),
+		getUpdateBusinessDetailsTask(
+			errorMessages,
+			status ?? '',
+			accountLink,
+			Number( currentDeadline ) ?? null,
+			pastDue ?? false
+		),
+		getReconnectWpcomTask( wpcomReconnectUrl ),
+		getDisputeResolutionTask( activeDisputes ),
+		getVerifyBankAccountTask(),
 	].filter( Boolean );
 };
 
